app/public/css/styles.css

```diff
@@ -1690,6 +1690,18 @@ button[data-balloon] {
   color: var(--color-grey);
   word-break: break-all;
   margin-bottom: 16px;
+  display: flex;
+}
+
+.address-number {
+  font-weight: bold;
+  word-break: normal;
+  margin-right: 15px;
+  user-select: none;
+  -moz-user-select: none;
```

Comment:
I would rather make a separate class `unselectable` for that since it's a generic property other elements may have and the classname would make clear in the html that the element is indeed unselectable